adagucserverEC/CDBFileScanner.cpp

```diff
@@ -85,13 +85,6 @@ int CDBFileScanner::createDBUpdateTables(CDataSource *dataSource, int &removeNon
     return 1;
   }
 
-  // Check if variable is in this file:
```

Comment:
Why is this no longer needed?

Reply:
Good poiint. It breaks the windshearprocessor, because the variable in windshearprocessor is renamed. I will check this.

Reply:
I have put it back, with a fix.

Since the previous layer renames and scans the variable, the next layer is unable to find that same variable for scanning. 

The cdfobject cache is now disabled when a variable is renamed.